src/foam/nanos/notification/email/DefaultPropertyEmailTemplateService.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
+  name: 'DefaultPropertyEmailTemplateService',
```

Comment:
I'd maybe call this something like EmailConfigEmailPropertyService since it's an EmailPropertyService that uses the EmailConfig.